src/Compilers/Core/Portable/Syntax/SyntaxAnnotation.cs

```diff
@@ -18,7 +18,7 @@ public sealed class SyntaxAnnotation : IObjectWritable, IObjectReadable, IEquata
         /// <summary>
         /// A predefined syntax annotation that indicates whether the syntax element has elastic trivia.
         /// </summary>
-        public static readonly SyntaxAnnotation ElasticAnnotation = new SyntaxAnnotation();
+        public static SyntaxAnnotation ElasticAnnotation { get; } = new SyntaxAnnotation();
```

Comment:
One of the reasons to prefer a property here over a field is Mono support.  The Mono support for cycles between static and instance constructors of the same time is less than that of the CLR.  In some cases code which executes successfully on CLR will stack overflow on Mono when there is a constructor cycle.  

In these cases when the static data is backed by a property this cycle is really easy to break.  On the other hand if it is backed by a field then it is not easy, and sometimes simply not possible.  There are already several cases where we have had to change static data initialization to support our Mono scenarios.